src/shm_ext.c

```diff
@@ -1920,7 +1920,8 @@ sr_shmext_oper_sub_check(sr_conn_ctx_t *conn, sr_mod_t *shm_mod, const struct ly
     for (i = 0; i < shm_mod->oper_get_sub_count; ++i) {
         valid = 0;
         xp_subs = (sr_mod_oper_get_xpath_sub_t *)(conn->ext_shm.addr + subs[i].xpath_subs);
-        for (j = 0; j < subs[i].xpath_sub_count; ++j) {
+        j = 0;
+        if (j < subs[i].xpath_sub_count) {
```

Comment:
The code is formatted this way to be the same as the other functions for other subscriptions. I do not care that `coverity` complains about it.